src/gobexport/exporter/config/brk2/kadastraleobjecten.py

```diff
@@ -92,6 +98,306 @@ def is_valid(aandeel: dict):
     return Fraction(a["teller"], a["noemer"]) > Fraction(b["teller"], b["noemer"])
 
 
+class KadastraleobjectenCsvFormat:
```

Comment:
Lots of duplicate code between this class and the BRK1 version. Especially the formatting code. Should be extracted to a separate class and/or module.

Reply:
Agreed. Only `KadastraleobjectenCsvFormat.get_format()` differs.